### DISCUSSION TITLE
Add filter error

### DISCUSSION THREAD

**Discussion by ku-zhen** [RafaelGB/obsidian-db-folder#317] [Q&A] [answered]
**Describe the bug**

After updating, adding filters will cause errors.

**To Reproduce**

**YAML**

citeKey: cadena_past_2016
entry: "[Past, Present, and Future of Simultaneous Localization and Mapping: Toward the Robust-Perception Age](zotero://select/items/@cadena_past_2016) "
DOI: 10.1109/TRO.2016.2624754
tags: Papers/SLAM
status: 
archive: 0
publishyear: 2016
create: 2022-05-06

**Steps to reproduce the behavior:**

1. Click on 'add filter'
2. See error

**Screenshots**

![image](https://user-images.githubusercontent.com/61906420/188152432-a9aead32-8cb9-4288-b083-91b3cac58355.png)

**Desktop (please complete the following information):**
 - OS:  win10
 - Version: obsidian 0.15.9

**Comment by RafaelGB** [comment]
Now you can not have an empty "contains" filter use is empty is not empty instead

**Comment by RafaelGB** [comment]
I could test your problem now. Added as a bug!

**Comment by RafaelGB** [comment, marked as answer]
https://github.com/RafaelGB/obsidian-db-folder/issues/322

**Comment by RafaelGB** [comment]
Sorry @wuzhen97 for the inconvenience, was fixed with v2.3.4
